ts/output/chtml.ts

```diff
@@ -47,7 +48,11 @@ import {unicodeChars} from '../util/string.js';
  * @template D  The Document class
  */
 export class CHTML<N, T, D> extends
-CommonOutputJax<N, T, D, CHTMLWrapper<N, T, D>, CHTMLWrapperFactory<N, T, D>, CHTMLFontData, typeof CHTMLFontData> {
+CommonOutputJax<
+  N, T, D,
+  ChtmlWrapper<N, T, D>, ChtmlWrapperFactory<N, T, D>, ChtmlWrapperClass<N, T, D>,
+  ChtmlCharOptions, ChtmlVariantData, ChtmlDelimiterData, ChtmlFontData, ChtmlFontDataClass
+> {
```

Comment:
I think it is worth explaining those in more detail, i.e., not just the template types.